src/parser/mod.rs

```diff
@@ -681,17 +681,49 @@ impl<'a> Parser<'a> {
 
     pub fn parse_truncate(&mut self) -> Result<Statement, ParserError> {
         let table = self.parse_keyword(Keyword::TABLE);
-        let table_name = self.parse_object_name(false)?;
+        let only = self.parse_keyword(Keyword::ONLY);
+
+        let table_names = self.parse_comma_separated(|p| p.parse_object_name(false))?;
+
+        // Unwrap is safe - the preceding parse  fails if there is not at least one table name
+        let table_name = table_names.first().unwrap().clone();
```

Comment:
```suggestion
        let Some(table_name) = table_names.first().clone() else {
               parser_err!(...)
        };
```
Can we use something like this format in any case? I think that would be safer if the semantics of the preceeding parse fails, or the code gets restructured etc in the future.